log-appender/src/main/java/io/phdata/pulse/log/HttpAppender.java

```diff
@@ -61,15 +72,32 @@
 
   public HttpAppender() {
     try {
-      hostname = java.net.InetAddress.getLocalHost().getHostName();
+      this.hostname = java.net.InetAddress.getLocalHost().getHostName();
     } catch (UnknownHostException e) {
       LogLog.error("Could not set hostname: " + e, e);
     }
+
+    this.verifiedSparkApplication = Util.isSparkApplication();
+    if (verifiedSparkApplication) {
+      try {
```

Comment:
I'd move these try/catch blocks into the Util methods and return a null if an exception is thrown.

Reply:
Done.

Reply:
Instead of returning null I'm returning the empty string. I think they are probably equivalent but I know for sure empty string won't break anything downstream.